Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,469 @@
+There are benefits to aerospikes but there are also disadvantages. The advantages are concerned with the region of optimal expansion. The aerospike nozzle can match the
+ambient pressure over a larger region and therefore can produce more thrust. However, the disadvantages are increased thermal loading, which is not terribly important here but is worht mentioning,
+and high design and fabrication complexity.
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
+
+\onecolumn
+\appendices{}
+\section{\textbf{Detailed Fall Timeline}}
+\centering
+\begin{enumerate}
+  \item Week 1
+  \begin{enumerate}
+    \item Introductions \& Fall plan
+  \end{enumerate}
+  \item Week 2
+  \begin{enumerate}
+    \item Beginner Project: Supersonic Nozzle Design
+    \item Test Stand Re-design Discussion
+  \end{enumerate}
+  \item Week 3
+  \begin{enumerate}
+    \item Beginner Project
+    \item Test Stand Re-design fabrication
+  \end{enumerate}
+  \item Week 4
+  \begin{enumerate}
+    \item Beginner Project testing
+  \end{enumerate}
+  \item Week 5
+  \begin{enumerate}
+    \item Preliminary CGT testing (just to familarize ourselves with the system and safety procedure)
+  \end{enumerate}
+  \item Week 6
+  \begin{enumerate}
+    \item Discussion of nozzle design, nozzle design, and printing
+  \end{enumerate}
+  \item Week 7
+  \begin{enumerate}
+    \item Nozzle Design and printing
+  \end{enumerate}
+  \item Week 8
+  \begin{enumerate}
+    \item Round 1 of CGT Testing
+  \end{enumerate}
+  \item Week 9
+  \begin{enumerate}
+    \item Round 2 of CGT Testing
+  \end{enumerate}
+  \item Week 10
+  \begin{enumerate}
+    \item Data Concatenation and sorting
+  \end{enumerate}
+  \item Week 11
+  \begin{enumerate}
+    \item Aerospike Dicussion and theory
+  \end{enumerate}
+  \item Week 12
+  \begin{enumerate}
+    \item Aerospike nozzle design and fabrication
+  \end{enumerate}
+  \item Week 13
+  \begin{enumerate}
+    \item Aerospike Preliminary testing, make design adjustments if necessary
+  \end{enumerate}
+  \item Week 14
+  \begin{enumerate}
+    \item Round 1 of Aerospike testing
+  \end{enumerate}
+  \item Week 15
+  \begin{enumerate}
+    \item Round 2 of Aerospike testing
+  \end{enumerate}
+  \item Week 16
+  \begin{enumerate}
+    \item Data Analysis
+  \end{enumerate}
+\end{enumerate}
+
+\newpage
+\section{Cold Gas Thruster Preliminary Bill of Materials}
+
+\begin{figure}[h]
+  \includegraphics[width = \linewidth]{figs/CGT-BoM.png}
+  \caption{This is the preliminary BoM that the propulsion team came up with last year. Most of the materials have already been acquired. But the propulsion system materials must still be purchased.}
+\label{fig: CGT BoM}
+\end{figure}
+\end{document}
```

Comment:
Add `\bibliography{your-bib-file-name-here}` before `\end{document}`
This will place the references for citations you might make elsewhere in the PDD